src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -977,7 +977,7 @@ public ChangelogCommand changelog() {
         return new ChangelogCommand() {
 
             /** Equivalent to the git-log raw format but using ISO 8601 date format - also prevent to depend on git CLI future changes */
-            public static final String RAW = "commit %H%ntree %T%nparent %P%nauthor %aN <%aE> %ai%ncommitter %cN <%cE> %ci%n%n%w(76,4,4)%s%n%n%b";
+            public static final String RAW = "commit %H%ntree %T%nparent %P%nauthor %aN <%aE> %ai%ncommitter %cN <%cE> %ci%n%n%w(0,4,4)%B%n%n";
```

Comment:
Can you add an item to the release notes (on the client plugin wiki page) that describes the change of responsibility?

I assume your plan is that the git client plugin will no longer truncate the first line of the commit message.  The git client plugin will deliver the entire first line of the commit message and rely on the git plugin to decide if it should be truncated (current behavior) or not.

Can you confirm that is your plan?

Reply:
Replaces #339

Reply:
Exactly, that is the plan